sig/sig.go

```diff
@@ -1,21 +1,42 @@
 package sig
 
-import "bytes"
+import (
+	"bytes"
+	"crypto/ecdsa"
+)
 
+// Hash is the result of SHA3_256
 type Hash [32]byte
 
 func (hash Hash) Equal(other Hash) bool {
 	return bytes.Equal(hash[:], other[:])
 }
 
-type Signature [65]byte
+type Signature []byte
```

Comment:
Why has this been changed? Using a static size is best, because then the value is stack bound.

Reply:
That was a mistake, it should be `[65]byte`. While I was trying out "crypto/ecdsa" directly, I thought I had to change that type to include `R`, `S` and a `Hash` since they didn't provide a verify for `[]byte`. In reality their `Sign` implementation uses `marshal` under the hood which means you need to use `unmarshal` for `Verify`, but it is not documented.